tailwind.config.js

```diff
@@ -9,6 +9,10 @@ module.exports = {
   theme: {
     extend: {
       ai: {},
```

Comment:
put the colors in here, again like the other acm projects
example:
`ai : { "blue": "#AEC6D8" }`